sdk/mobilenetwork/Azure.ResourceManager.MobileNetwork/src/Generated/Models/MobileNetworkCoreNetworkType.cs

```diff
@@ -7,12 +7,14 @@
 
 namespace Azure.ResourceManager.MobileNetwork.Models
 {
-    /// <summary> The core network technology generation (5G core or EPC / 4G core). </summary>
-    public enum CoreNetworkType
+    /// <summary> The core network technology generation (5G core, EPC / 4G core or EPC / 4G + 5G core). </summary>
+    public enum MobileNetworkCoreNetworkType
     {
         /// <summary> 5G core. </summary>
-        FiveGC,
+        FiveGC = "5GC",
```

Comment:
this does not seem correct. As far as I know our generator will not do this.
Could you run the generate command again? This should be fixed by the generator.